src/loki/search.cc

```diff
@@ -22,6 +23,23 @@ template <typename T> inline T square(T v) {
   return v * v;
 }
 
+bool is_search_filter_triggered(const DirectedEdge* edge,
+                                const Location::SearchFilter& search_filter) {
+  // check if this edge matches any of the exclusion filters
+  uint32_t road_class = static_cast<uint32_t>(edge->classification());
+  uint32_t min_road_class = static_cast<uint32_t>(search_filter.min_road_class_);
+  uint32_t max_road_class = static_cast<uint32_t>(search_filter.max_road_class_);
+
+  if ((road_class > min_road_class || road_class < max_road_class) ||
```

Comment:
its confusing that this is exclusive like the other filters. for some reason i think of min and max road classes as a bounds of what IS ALLOWED. i think it would be ok to have this part of the filter mean exclude road classes outside of this range for the sake of everyone's sanity

Reply:
specifically i was thinking `rc < min || max < rc` would be filtered